### PR TITLE
470 filter by event name

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Add filtering by event name to eventstable
image

## Checklist
- [x] Cypress E2E tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/actions/EventName.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useValues } from 'kea'
 import { userLogic } from 'scenes/userLogic'
 
 export function EventName({ value, onChange }) {
-    const { eventNamesGrouped } = useValues(userLogic)
+    const { eventNamesGrouped, eventNames } = useValues(userLogic)
```
Comment (EDsCODE) [inline frontend/src/scenes/actions/EventName.js:7]: Small mismatch between these two lists throughout the component. The disabled prop only relies on eventNames. The custom event prompt shows up even when there are events you can use to filter which makes using the filter on the event table somewhat confusing. Also, now you can select a default event on the action step event filter. Not sure if this is intentional or not

**Review by EDsCODE — COMMENTED** [review]
Works well. Small consideration/change below

File: frontend/src/scenes/actions/EventName.js (hunk 1 above)
Comment (timgl) [reply frontend/src/scenes/actions/EventName.js]: Fixed! Ready for re-qa

Comment (timgl) [reply frontend/src/scenes/actions/EventName.js]: PS you're currently also able to select default PostHog events. I think that's fine, there might be a usecase for having an action on pageleaves for example

**Review by EDsCODE — APPROVED** [review]
Not able to use default events on action step (opposite of what you suggested above). 

Otherwise good to go

**Pull request merged**